projects/fsastorefrontstyles/scss/components/account/_premium-calendar.scss

```diff
@@ -1,27 +1,136 @@
 @import '../../theme/fs/variables';
 
 %cx-fs-premium-calendar {
+  .container-fluid {
+    @include media-breakpoint-down(md) {
+      padding: 0;
+      display: grid;
+      grid-template-columns: repeat(2, 1fr);
+      grid-gap: 1.25rem;
+    }
+    @include media-breakpoint-down(sm) {
+      grid-template-columns: 100%;
+    }
+    .row {
+      @include media-breakpoint-down(md) {
+        display: grid;
+        place-items: center;
+        text-align: center;
+        margin: 0;
+      }
+    }
+    .border:nth-last-of-type(1) {
+      @include media-breakpoint-down(md) {
+        padding: 1.25rem 0.94rem;
+        height: 5.875rem;
+        font-size: var(--cx-h6-font-size);
+        color: var(--cx-color-6);
+      }
+    }
+  }
   .premium-data-row {
     background-color: var(--cx-color-4);
     border-top: none;
-    padding: 26px 0;
+    padding: 1.625rem 0;
+    @include media-breakpoint-down(md) {
+      padding: 0;
```

Comment:
Check if padding override is needed...

Reply:
If I don't use that, I have problem with borders because of the padding:

![Screenshot 2020-12-03 at 13 55 56](https://user-images.githubusercontent.com/75082466/101020967-88eff000-356f-11eb-8fe4-470d7b56642a.png)